pandas/io/formats/printing.py

```diff
@@ -272,7 +272,7 @@ class TableSchemaFormatter(BaseFormatter):
 
 
 def format_object_summary(obj, formatter, is_justify=True, name=None,
-                          indent_for_name=True):
+                          indent_for_name=True, line_break_each_value=False):
```

Comment:
Are there any tests that hit this being False? AFAICT added tests always assume a line break for each entry, no?

Reply:
This function wasn't itself tested before, so I haven't made tests for this specifically. I think this is ok, as all the different repr's are tested and that's what's important, IMO.

Of course, ``line_break_each_value=False`` is used in all repr's except the one for MultiIndex, so that is just the status quo before this PR.